lib/bootsnap/cli.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+require 'bootsnap'
+require 'optparse'
+require 'fileutils'
+
+module Bootsnap
+  class CLI
+    unless Regexp.method_defined?(:match?)
+      module RegexpMatchBackport
+        refine Regepx do
```

Comment:
Since it's just a CLI and the process is thrown away, it might be enough to define the method directly? Or is the code actually loaded? I thought it was just parsed and the iseq cached.

Reply:
I'd rather avoid doing it. I'll likely add some tests for that class, so it's better if it doesn't monkey patch Regexp.